python/hail/expr/types.py

```diff
@@ -310,14 +290,9 @@ def _repr(self):
 
 
 class TBoolean(Type):
-    """
-    Hail type corresponding to bool.
-
-    .. include:: ../_templates/hailType.rst
-
-    - `expression language documentation <types.html#boolean>`__
-    - in Python, these are represented natively as Python booleans (i.e. ``True`` and ``False``)
+    """Hail type for boolean (``True`` or ``False``) values.
```

Comment:
pedantic, but boolean is named for a person, so I kind of feel it should be capitalized in English text.

Reply:
Really? Happy to change but scientific units like newton and joule aren't usually capitalized

Reply:
Hmm. english.se [seems to prefer capitalization of Boolean](https://english.stackexchange.com/questions/4481/should-the-word-boolean-be-capitalized) in particular, though there's a telling comment about the decapitalization of words over time re: Abelian.

(Aside: I had no idea that newton was meant to be lower-case, I've always capitalized it in Physics class following the same logic as Boolean!)

I don't have a strong feeling on Boolean either way :)